roofit/histfactory/inc/RooStats/HistFactory/FlexibleInterpVar.h

```diff
@@ -56,6 +56,11 @@ namespace HistFactory{
     virtual void printMultiline(std::ostream& os, Int_t contents, Bool_t verbose = kFALSE, TString indent = "") const;
     virtual void printFlexibleInterpVars(std::ostream& os) const;
 
+    const RooListProxy& variables() const;
+    double nominal() const;        
+    const std::vector<double>& low() const;
+    const std::vector<double>& high() const;    
```

Comment:
These clash with local variables in the implementation, e.g. FlexibleInterpVar.cxx:56

Reply:
and the ones in lines 91, 142